### DISCUSSION TITLE
Add file directories.

### DISCUSSION THREAD

**Discussion by harishbit** [Pradumnasaraf/open-source-with-pradumna#267] [Q&A] [answered]
Hey, I was thinking to add file directories in readme file. What's your opinion ?

**Comment by Pradumnasaraf** [comment]
I didn't get what do you mean

**Comment by harishbit** [comment]
I was thinking to add file directory somthing like this. ![image](https://github.com/Pradumnasaraf/open-source-with-pradumna/assets/111338069/96eaada2-857b-4476-8a81-dd7ec9eb7aa5) (This is just for representation)

**Reply by Pradumnasaraf** [reply, marked as answer]
I don't think we need this because we have only one HTML file and no CSS file. We host via Markdown files that can't be restructured.

**Reply by harishbit** [reply]
Ok. No problem.
